src/CMakeLists.txt

```diff
@@ -3,10 +3,10 @@
 #
 
 set(SOURCES MultiBodyGraph.cpp MultiBody.cpp MultiBodyConfig.cpp
-  FK.cpp FV.cpp FA.cpp Jacobian.cpp ID.cpp IK.cpp IS.cpp FD.cpp EulerIntegration.cpp
+  FK.cpp FV.cpp FA.cpp Jacobian.cpp ID.cpp IK.cpp IS.cpp FD.cpp NumericalIntegration.cpp
```

Comment:
I am not really use to deal with deprecation stuff but if `EulerIntegration.cpp` is deleted then the program just won't compile at all for old style and there is no need to mark it deprecated. Should'nt the file stay alive for few release iteration before it is definitively deleted?